modules/order/src/Entity/Order.php

```diff
@@ -463,6 +494,21 @@ public static function baseFieldDefinitions(EntityTypeInterface $entityType) {
       ->setLabel(t('Data'))
       ->setDescription(t('A serialized array of additional data.'));
 
+    $fields['billing_address'] = BaseFieldDefinition::create('entity_reference')
```

Comment:
We are referencing the billing profile, not the billing address. You'll need to correct the field name and getters/setters accordingly.